### PR TITLE
Translated missing

### PR DESCRIPTION
**Pull request by SalahAdDin**
Please, regenerate this .po file and i will translate it :D

### CODE REVIEW

File: wagtail/wagtaildocs/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -127,11 +127,11 @@ msgstr "Editando"
 #, fuzzy
```
Comment (kaedroho) [inline wagtail/wagtaildocs/locale/es/LC_MESSAGES/django.po]: Need to get rid of this "fuzzy" attribute

**Comment by kaedroho** [comment]
Thanks! Just remove that fuzzy marker and I'll merge

**Comment by kaedroho** [comment]
Merged in: b125ce1142b04e214c24b64d1b74efbaa60f18ff

Thanks!

**Comment by SalahAdDin** [comment]
Oh, thank you :D

**Comment by SalahAdDin** [comment]
Man, i did many fix in the some locale files and then i finished the translation in Transifex, if you want you can merge both changes and put in the main project, and please, put a reviewer user in Transifex for Es locale files.

**Pull request closed without merging**